src/rpc/rawtransaction.cpp

```diff
@@ -773,23 +774,23 @@ UniValue signrawtransaction(const JSONRPCRequest& request)
         }
         const CScript& prevPubKey = coins->vout[txin.prevout.n].scriptPubKey;
         const CAmount& amount = coins->vout[txin.prevout.n].nValue;
-
+	const CTransactionRef& coinbaseTx = chainActive.Tip()->coinbase;
```

Comment:
My approach to this is wrong. There is NO way we can use `signrawtransaction` with a transaction that spends an `OP_BV` output. Only miners can create valid transactions that spend OP_BV outputs. So it is useless to call `signrawtransaction` if one of the transactions prevout scripts is a `OP_BV` output. Probably should just fail the rpc command.